commcare_connect/templates/account/base.tailwind.html

```diff
@@ -0,0 +1,57 @@
+{% load static i18n %}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>{% block title %}CommCareConnect - {% block head_title %}{% endblock head_title %}{% endblock title %}</title>
+    {% load static %}
+    {% block css %}
+      <link rel="stylesheet" href="{% static 'bundles/css/tailwind.css' %}">
+      <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@400;500;600;700&display=swap" rel="stylesheet">
+      <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/aquawolf04/font-awesome-pro@5cd1511/css/all.css" />
+    {% endblock %}
+    {% block javascript %}
+      <script src="{% static 'bundles/js/vendors-bundle.js' %}" defer></script>
+      <script src="{% static 'bundles/js/tailwind-bundle.js' %}" defer></script>
+    {% endblock javascript %}
+  </head>
+  <body class="flex flex-col justify-center pl-16 overflow-x-hidden bg-stone-100 font-work-sans">
+    {% block content %}
```

Comment:
Why is this a block? It looks like usages of this template should be using the `inner` block, not this one, and it is not extending an existing template so can't be that it is overwriting parent values

Reply:
Fixed. [4c50688](https://github.com/dimagi/commcare-connect/pull/581/commits/4c50688d3fbce21d8ea3bd0174185aef5461c024)